packages/dashboard/src/views/Holdings.js

```diff
@@ -190,6 +193,53 @@ const Holdings = ({
   // todo move this to only update on account change
   const accountSmallStats = renderSmallStats()
 
+  const renderTableData = () => {
+    // check if we have account  data
+    if (historicQuotes['1d'] && historicQuotes['all']) {
```

Comment:
what happens if the user is logged out or we dont have data? Is it a blank card/table or nothing

Reply:
just a blank table